src/Console/Helper/Question/AlwaysReturnKeyOfChoiceQuestion.php

```diff
@@ -118,7 +118,11 @@ private function prepareChoices()
         foreach ($this->choiceViews as $choiceView) {
             $label = $choiceView->label;
             if ($choiceView->data != $choiceView->value) {
-                $label .= ' (<comment>'.$choiceView->data.'</comment>)';
+                try {
```

Comment:
Instead of relying on an ErrorException, I'd really prefer having a check for "can be turned into a string". I'd even guess we're not the first ones to need this check; there may even be something in the Symfony code base for this, but otherwise we can generalize and say that stringable means scalar or an object with a __toString() method.

Reply:
Fixed, based on answers to this SO question https://stackoverflow.com/q/5496656/2034213 and grepping through Symfony sources (and other packages in `vendor` directories of my recent projects). There seems to be no standard utility function for this in Symfony, so I took this
https://github.com/symfony/symfony/blob/ad6e888a6f90e4681d57901584a7967aff332582/src/Symfony/Component/HttpKernel/Log/Logger.php#L88